metadata-service/plugin/src/main/java/com/datahub/plugins/common/PluginConfig.java

```diff
@@ -0,0 +1,17 @@
+package com.datahub.plugins.common;
+
+import java.util.Map;
+import java.util.Optional;
+
+
+public interface PluginConfig {
+  public PluginType getType();
```

Comment:
Nit: let's add javadocs to each thing here

Reply:
Done